docs/en/reference/query-builder.rst

```diff
@@ -368,6 +368,26 @@ or QueryBuilder instances to one of the following methods:
         ->orderBy('field', 'DESC')
         ->setMaxResults(100);
 
+WITH-Clause
```

Comment:
Let's replace this with "Common Table Expressions". I think it's important to document the usage of parameters in CTEs.